force-app/main/default/classes/OpportunityService.cls

```diff
@@ -0,0 +1,30 @@
+public with sharing class OpportunityService {
+
+    public static void setOppValuesFromTasks(Map<Id,Task> newTasksMap, Map<Id,Task> oldTasksMap )
+    {
+        //TODO: if(Triggers.Stop)
+
+        Set<Id> oppIds = new Set<Id>();
+
+        for (Task t : newTasksMap.values()) 
```

Comment:
`t` isn't a great variable name - I'd try to use something that reflects it is a task, such as `newTask`, `candidateTask`.